README.md

```diff
@@ -320,6 +322,85 @@ i18n.offChangeLocale(fn);
 i18n.onceChangeLocale(fn);
 ```
 
+## Pluralization
+
+With this package, you can translate with pluralization. To do that you need to use the pipe separator `|` in your locale translations, like in the example below:
+
+```yml
+_locale: 'en'
+phone: 'zero phones | one phone | two phones | {count} phones'
+```
+
+Then just add the `count` argument to the `getTranslation` options:
+
+```html
+<p>{i18n.getTranslation('phone', { count: 0 })}</p>
+<p>{i18n.getTranslation('phone', { count: 1 })}</p>
+<p>{i18n.getTranslation('phone', { count: 2 })}</p>
+<p>{i18n.getTranslation('phone', { count: 3 })}</p>
+<p>{i18n.getTranslation('phone', { count: 1000 })}</p>
+```
+
+Output:
+
+```html
+<p>zero phones</p>
+<p>one phone</p>
+<p>two phones</p>
+<p>3 phones</p>
+<p>1000 phones</p>
+```
+
+### Custom rules
+
+Some languages (e.g. from the Slavic group) may have more complicated pluralization rules. It is possible to define custom pluralization rules to handle such cases. To do that add the `pluralizationRules` property to `i18n.setOptions`:
+
+```js
+i18n.setOptions({
+  // Key - language to applye the rule for, e.g 'pl'
```

Comment:
```suggestion
  // Key - language in which the rule should be applied, e.g. 'en'
```

Reply:
I think the original wording is more accurate (except for the typo ofc). Your wording suggest the rule should be applied **_in_** the language, whereas the rule should be applied **_to_** the language, e.g. the rule should be applied **_to_** English, not **_in_** English.

Also, I think we should stick to the `pl` as this is the example I go with, in this section.